src/ncp/ncp_base_mtd.cpp

```diff
@@ -2877,6 +2909,35 @@ void NcpBase::HandleEnergyScanResult(otEnergyScanResult *aResult)
     }
 }
 
+void NcpBase::HandleJoinerCallback_Jump(otError aError, void *aContext)
```

Comment:
Missing `#if OPENTHREAD_ENABLE_JOINER` around the `HandleJoinerCallback_Jump()` and `HandleJoinerCallback()`,